packages/web/src/scss/components/FileUploader/index.html

```diff
@@ -440,30 +455,173 @@ <h3 id="attachments-fileUploadDisabled" hidden>Attachments</h3>
   </div>
 </section>
 <section class="docs-Section">
-  <h2 class="docs-Heading">Input with Attachment</h2>
+  <h2 class="docs-Heading">
+    Attachment
+    <span class="Tag Tag--medium Tag--subtle Tag--warning">Visual demo only</span>
+  </h2>
 
-  <div class="docs-FormFieldGrid">
-    <!-- FileUploader: start -->
-    <div class="FileUploader" data-spirit-toggle="fileUploader">
-      <!-- FileUploaderAttachment template: start -->
-      <template data-spirit-snippet="item">
-        <li class="FileUploaderAttachment" data-spirit-populate-field="item">
+  <!-- FileUploader: start -->
+  <div class="FileUploader" data-spirit-toggle="fileUploader">
+
+    <!-- FileUploaderList: start -->
+    <h3 id="attachments-examples" hidden>Attachments</h3>
+    <ul class="FileUploaderList" aria-labelledby="attachments-examples" data-spirit-element="list">
+
+      <!-- FileUploaderAttachment: start -->
+      <li class="FileUploaderAttachment">
+        <svg width="24" height="24" aria-hidden="true">
+          <use xlink:href="/icons/svg/sprite.svg#file" />
+        </svg>
+        <span class="FileUploaderAttachment__name">
+          <span class="text-truncate">Document.pdf</span>
+        </span>
+        <button type="button" class="FileUploaderAttachment__action">
+          <span class="accessibility-hidden">Remove</span>
           <svg width="24" height="24" aria-hidden="true">
-            <use xlink:href="/icons/svg/sprite.svg#file" />
+            <use xlink:href="/icons/svg/sprite.svg#close" />
+          </svg>
+        </button>
+      </li>
+      <!-- FileUploaderAttachment: end -->
+
+      <!-- FileUploaderAttachment: start -->
+      <li class="FileUploaderAttachment">
+        <span class="FileUploaderAttachment__image">
+          <img src="http://placekitten.com/200/300" alt="" />
```

Comment:
Just for consistency: Can we please use only one service for example images? In https://spirit-design-system-demo.netlify.app/src/scss/helpers/images/ we are using picsum.photos. I think we should pick only one service for these examples.

Reply:
<img width="328" alt="obrazek" src="https://github.com/lmc-eu/spirit-design-system/assets/5614085/b8cae488-e837-408d-9845-dcd80377b790">